src/test/java/tests/RegisterTest.java

```diff
@@ -0,0 +1,68 @@
+package tests;
+
+import common.helpers.DataHelper;
+import models.Account;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.RegisterPage;
+
+public class RegisterTest extends BaseTest {
+    HomePage homePage = new HomePage();
+    RegisterPage registerPage = new RegisterPage();
+    String email = DataHelper.getRandomEmail();
+    String expectedGeneralErrorMsg = "There're errors in the form. Please correct the errors and try again.";
+
+    @Test(description = "User can create new account")
+    public void TC007() {
+        Account account = new Account().getRandomAccount();
+
+        homePage.gotoRegisterPage();
+        registerPage.register(account);
+
+        String actualSuccessfulMsg = registerPage.getSuccessfulRegisterMsg();
+        String expectSuccessfulMsg = "Thank you for registering your account";
+
+        Assert.assertEquals(actualSuccessfulMsg, expectSuccessfulMsg, "Successful message is incorrect");
+    }
+
+    @Test(description = "User can't create account with Confirm Password is not the same with Password")
+    public void TC010() {
+        String password = "987654321";
+        String invalidConfirmPassword = "123456789";
+        String pid = DataHelper.getRandomPID();
+
+        Account account = new Account(email, password, invalidConfirmPassword, pid);
+
+        homePage.gotoRegisterPage();
+        registerPage.register(account);
+
+        String actualGeneralErrorMsg = registerPage.getGeneralErrorMsg();
+
+        Assert.assertEquals(actualGeneralErrorMsg, expectedGeneralErrorMsg, "General error message is incorrect");
+    }
+
+    @Test(description = "User can't create account while Password and PID fields are empty")
+    public void TC011() {
+        String password = "";
```

Comment:
don't need to define var for a blank.

Reply:
Thanks for your review! I fixed this mistake. Please review it